app/views/trips/dashboard.erb

```diff
@@ -0,0 +1,121 @@
+</div>
+
+<div class="trips-dashboard-average">
+<h2>Average Duration of a Ride</h2>
+<p class="trip-dashboard-average"><%= @trips.average_duration_of_ride %></p>
+</div>
+
+<div class="trips-dashboard-max">
+<h2>Longest Ride</h2>
+<p><%= @trips.longest_ride %></p>
+</div>
+
+<div class="trips-dashboard-min">
+<h2>Shortest Ride</h2>
+<p><%= @trips.shortest_ride %></p>
+</div>
+
+<div class="trips-dashboard-most-rides-starting">
+<h2>Station with Most Rides Starting There</h2>
+<p><%= @trips.most_rides_starting %></p>
+</div>
+
+<div class="trips-dashboard-most-rides-ending">
+<h2>Station with Most Rides Ending There</h2>
+<p><%= @trips.most_rides_ending %></p>
+</div>
+
+<div class="trips-dashboard-rides-per-year">
+<h2>Rides Per Month Each Year</h2>
+<p><%= @trips.rides_per_year(2013) %><br?></p>
+  <ul>
+    <li>January: <%= @trips.rides_per_month(1) %></li>
+    <li>February: <%= @trips.rides_per_month(2) %></li>
+    <li>March:<%= @trips.rides_per_month(3) %></li>
+    <li>April: <%= @trips.rides_per_month(4) %></li>
+    <li>May: <%= @trips.rides_per_month(5) %></li>
+    <li>June: <%= @trips.rides_per_month(6) %></li>
+    <li>July: <%= @trips.rides_per_month(7) %></li>
+    <li>August: <%= @trips.rides_per_month(8) %></li>
+    <li>September: <%= @trips.rides_per_month(9) %></li>
+    <li>October: <%= @trips.rides_per_month(10) %></li>
+    <li>November: <%= @trips.rides_per_month(11) %></li>
+    <li>December: <%= @trips.rides_per_month(12) %></li>
+  </ul>
+<p><%= @trips.rides_per_year(2014) %><br?></p>
+  <ul>
+    <li>January: <%= @trips.rides_per_month(1) %></li>
+    <li>February: <%= @trips.rides_per_month(2) %></li>
+    <li>March:<%= @trips.rides_per_month(3) %></li>
+    <li>April: <%= @trips.rides_per_month(4) %></li>
+    <li>May: <%= @trips.rides_per_month(5) %></li>
+    <li>June: <%= @trips.rides_per_month(6) %></li>
+    <li>July: <%= @trips.rides_per_month(7) %></li>
+    <li>August: <%= @trips.rides_per_month(8) %></li>
+    <li>September: <%= @trips.rides_per_month(9) %></li>
+    <li>October: <%= @trips.rides_per_month(10) %></li>
+    <li>November: <%= @trips.rides_per_month(11) %></li>
+    <li>December: <%= @trips.rides_per_month(12) %></li>
+  </ul>
+<p><%= @trips.rides_per_year(2015) %><br?></p>
+  <ul>
+    <li>January: <%= @trips.rides_per_month(1) %></li>
+    <li>February: <%= @trips.rides_per_month(2) %></li>
+    <li>March:<%= @trips.rides_per_month(3) %></li>
+    <li>April: <%= @trips.rides_per_month(4) %></li>
+    <li>May: <%= @trips.rides_per_month(5) %></li>
+    <li>June: <%= @trips.rides_per_month(6) %></li>
+    <li>July: <%= @trips.rides_per_month(7) %></li>
+    <li>August: <%= @trips.rides_per_month(8) %></li>
+    <li>September: <%= @trips.rides_per_month(9) %></li>
+    <li>October: <%= @trips.rides_per_month(10) %></li>
+    <li>November: <%= @trips.rides_per_month(11) %></li>
+    <li>December: <%= @trips.rides_per_month(12) %></li>
+  </ul>
+<p><%= @trips.rides_per_year(2016) %><br?></p>
```

Comment:
this snippet here is a great candidate for a partial - since the html here and code is pretty similar (except for the year). we can re-use this piece of html by putting it in a partial 👍 if you want to dive into that further and have questions, let me know!